configs/openmrs_config/ampathformstranslations/cervical_cancer_screening_km.json

```diff
@@ -9,7 +9,7 @@
       "Menstrual History": "ប្រវត្តិនៃការមករដូវ",
       "Menstrual Bleeding History": "ប្រវត្តិនៃការធ្លាក់ឈាមរដូវ",
       "Risk Factors" :"កត្តាហានិភ័យ",
-      "Cervical Cancer": "ជំងឺមហារីកមាត់ស្បូន",
+      "Cancer": "Cancer",
```

Comment:
@kazlaw should we consider adding the translation in the followup commit?

Reply:
Hi @kdaud  all the tickets currently have pending translations. I will assign them to you as a reviewer when they are ready. The progress is reported on the ticket comments in Jira for now

Reply:
The Khmer translation was added 

![cc-tab](https://github.com/openmrs/ozone-distro-cambodia/assets/36139599/aee9b4b4-275a-405a-8760-5aea780d1432)

Reply:
![cctab-big](https://github.com/openmrs/ozone-distro-cambodia/assets/36139599/8086fd2b-4424-4d3a-9a53-d643394f3a5a)